src/ui/components/CreateIdentifier/components/IdentifierStage3.tsx

```diff
@@ -20,6 +24,31 @@ const IdentifierStage3 = ({
 }: IdentifierStageProps) => {
   const dispatch = useAppDispatch();
   const [alertIsOpen, setAlertIsOpen] = useState(false);
+  // @TODO - sdisalvo: This is a temporary fix to get the identifier created.
+  // We'll need to work out a proper way to get 'ourIdentifier'.
+  const identifiersData = useAppSelector(getIdentifiersCache);
+  const ourIdentifier = identifiersData.filter(
+    (identifier) => identifier.method === IdentifierType.KERI
+  )[0]?.id;
+  const otherIdentifierContacts: ConnectionShortDetails[] =
```

Comment:
hmm, this will run on every render (alert open close) maybe should be a state variable and just sort it whenever `state.selectedConnections` changes